app/main/views/audits.py

```diff
@@ -83,23 +90,43 @@ def list_audits():
     if object_type:
         if object_type not in AUDIT_OBJECT_TYPES:
             abort(400, 'invalid object-type supplied')
-        if not object_id:
-            abort(400, 'object-type cannot be provided without object-id')
```

Comment:
As far as I remember one of the reasons it was disallowed was that these requests ended up generating an expensive query plan due to not using the index. And filtering by audit event type seemed like a reasonable alternative for things you'd look for with a bare object-type filter. Do we need this combination of filters for something now?